packages/frontend/web/components/Header/Nav/ReaderRevenueLinks.tsx

```diff
@@ -120,58 +97,66 @@ export const ReaderRevenueLinks: React.FC<{
+    const requiredLinkStyle = noResponsive ? link : cx(link, hiddenUntilTablet);
+
     return (
         <AsyncClientComponent f={shouldShow}>
             {({ data }) => (
                 <>
                     {data && (
                         <>
-                            <div className={readerRevenueLinks}>
-                                <div className={message}>
-                                    Support The Guardian
-                                </div>
-                                <div
-                                    className={cx(
-                                        subMessage,
-                                        hiddenUntilTablet,
-                                    )}
-                                >
-                                    Available for everyone, funded by readers
-                                </div>
-                                <a
-                                    className={cx(link, hiddenUntilTablet)}
-                                    href={urls.contribute}
-                                    data-link-name={`${dataLinkNamePrefix}contribute-cta`}
-                                >
-                                    Contribute <ArrowRightIcon />
-                                </a>
-                                <a
-                                    className={cx(link, hiddenUntilTablet)}
-                                    href={urls.subscribe}
-                                    data-link-name={`${dataLinkNamePrefix}subscribe-cta`}
-                                >
-                                    Subscribe <ArrowRightIcon />
-                                </a>
-                                <a
-                                    className={cx(link, hiddenFromTablet, {
-                                        [hidden]: edition !== 'UK',
-                                    })}
-                                    href={urls.support}
-                                    data-link-name={`${dataLinkNamePrefix}support-cta`}
-                                >
-                                    Support us <ArrowRightIcon />
-                                </a>
-                                <a
-                                    className={cx(link, hiddenFromTablet, {
-                                        [hidden]: edition === 'UK',
-                                    })}
-                                    href={urls.contribute}
-                                    data-link-name={`${dataLinkNamePrefix}contribute-cta`}
-                                >
-                                    Contribute <ArrowRightIcon />
-                                </a>
+                            {/* <div className={readerRevenueLinks}> */}
+                            <div
+                                className={cx(message, {
+                                    [hiddenUntilTablet]: !noResponsive,
```

Comment:
Could this be handled with a wrapper rather than adding complexity to the two `classNames` Or even handle `noResponsive` stuff like 102 but all in one place? It's a bit scattered through the component and it makes it a little hard to reason with

Reply:
Made this change